src/test/java/greencity/service/impl/PhotoServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package greencity.service.impl;
+
+import greencity.GreenCityApplication;
+import greencity.entity.Photo;
+import greencity.repository.PhotoRepo;
+import greencity.service.PhotoService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Optional;
+
+import static org.mockito.Mockito.when;
+
+
+import static org.junit.Assert.*;
+
+@RunWith(MockitoJUnitRunner.class)
+@SpringBootTest(classes = GreenCityApplication.class)
```

Comment:
you should use @RunWith(MockitoJUnitRunner.class) instead of @SpringBootTest(classes = GreenCity Application.class)